src/test/java/com/google/jenkins/plugins/computeengine/StringJsonServiceAccountConfig.java

```diff
@@ -69,7 +74,7 @@ private String writeJsonKeyToFile(JsonKey jsonKey) throws IOException {
 
     @Override
     public com.google.jenkins.plugins.credentials.oauth.JsonServiceAccountConfig.DescriptorImpl getDescriptor() {
-        return (com.google.jenkins.plugins.credentials.oauth.JsonServiceAccountConfig.DescriptorImpl) Jenkins.getInstance()
+        return (com.google.jenkins.plugins.credentials.oauth.JsonServiceAccountConfig.DescriptorImpl) Jenkins.get()
```

Comment:
You might want to import `com.google.jenkins.plugins.credentials.oauth.JsonServiceAccountConfig` to cut down on a lot of the cruft in this file.

Reply:
Good catch. done